jenkins/gotest-dockerized.sh

```diff
@@ -36,7 +36,7 @@ mkdir -p "${HOST_ARTIFACTS_DIR}"
 # provided must be resolvable on the *HOST*, not the container.
 
 # default to go version 1.5 image tag
-KUBEKINS_TAG="1.5-v20161109-d9a98f2"
+KUBEKINS_TAG="1.5-v20161205-d664d14"
```

Comment:
I made the PR in https://github.com/kubernetes/test-infra/pull/1299 do that so we can get this in before the golint fixes on master